docs/src/pages/components/masonry/BasicMasonry.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+import { divData } from './demoData';
```

Comment:
Let's add the date in the same file. See how it is done for example with the `ImageList` demos. Add the date after the component so that it is not distracting the user when opening the demo.